aiida/tools/data/cif.py

```diff
@@ -162,6 +162,7 @@ def refine_inline(node):
 
     .. note:: can be used as inline calculation.
     """
+    # pylint: disable=unsubscriptable-object
```

Comment:
which line does this come from? I know there is a bug in `pylint` in combination with Python 3.9 that throws these as false positives

Reply:
done